response_operations_ui/templates/create-collection-exercise.html

```diff
@@ -0,0 +1,60 @@
+{% extends "layouts/spoke.html" %}
+{% block page_title %} Create collection exercise {% endblock %}
+{% block main %}
+<div role="main" id="main" class="page__main">
+  <div class="spoke__header">
+    <div class="spoke__header__title">
+      <h1 class="jupiter">Create collection exercise</h1>
+    </div>
+  </div>
+  <div class="spoke__body">
+    <form action="" method="post">
+      {{ form.csrf_token }}
+      <div class="field u-mb-l">
+        <label class="label" for="Survey">
+          Survey:
+          <span class="u-vh">Current value: </span>
+          <span class="mars">
+                {{ survey_ref }} {{ short_name }}
+              </span>
+        </label>
+        <br>
+      {% if errors %}
+      <div class="panel panel--simple panel--error u-mb-l">
+        <div class="panel__body" id="save-error">
+          <h2 id='error-creating-ce-message' class="venus">Error creating collection exercise</h2>
+          <p class="mars">Please use a period that is not in use by any collection exercise for this survey</p>
+        </div>
+      </div>
+      {% endif %}
+        <div class="field u-mb-l">
+          <label class="label" for="period">
+            Period
+          </label>
+          {{ form.period(class_="input input--text", id='period', required=required, maxlength=6, value=period) }}
```

Comment:
May want to add some more validation, when I type a letter and save I get sent to error 500 page

Reply:
Same kind of error with the 'shown to respondent' if I copy and paste over a certain amount of characters I get a 500

Reply:
yeah my bad i set the field limit to be 100 when in the database it has a max of 50, ill change that

Reply:
Should we have validation in the html like this if we're using flaskform for it?

Reply:
:+1: Having both may the best way to go, the problem with the validation inline like this is that users can edit the html and submit stuff they shouldn't be able to, so we should always have the server side validation at least